spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+require 'support/shared_contexts/shared_authenticated.rb'
+
+describe Api::V1::BooksController, type: :controller do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    context 'When fetching all the users rents' do
+      before do
+        @book = create(:book)
+        @rent = Rent.create(
```

Comment:
Same here. do: `let!(:rent) { create(:rent, user: user, book: book) }`

Reply:
Wait. Why are you creating a rent? And if you create one, use a factory

Reply:
> Wait. Why are you creating a rent? And if you create one, use a factory

I don' know. Deleted it.